utils/__mocks__/zip.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Mock implementation of the zip module
+ */
+
+export const createCompressedStream = jest.fn(() => ({
```

Comment:
generally we don't need this kind of mocks, simply call `jest.mock(<module path>)` will transform every exported functions form that module into a mock (jest.fn)